session01/break_me.py

```diff
@@ -1,3 +1,38 @@
+<<<<<<< HEAD
```

Comment:
This is cruft left over from a "merge conflict: form git.

When git tries to merge two versions, and it can't tell which one you want, it puts both in, with this stuff marking it.

You need to go in by hand and save the version of the code you want, and remove the merge markers.

Then you can commit that change, and all will be well.